internal/controller/bootstrap/init.go

```diff
@@ -468,6 +468,27 @@ func (b *Bootstrap) CreateOrUpdateFromYaml(yamlContent []byte, alwaysUpdate ...b
 				obj.Object["spec"].(map[string]interface{})["config"] = sub.Object["spec"].(map[string]interface{})["config"]
 			}
 			update = !equality.Semantic.DeepEqual(sub.Object["spec"], obj.Object["spec"])
+		} else if gvk.Kind == "Certificate" {
```

Comment:
Hi @qpdpQ 

I am thinking that CS operator will [never "update" a certificate resource](https://github.com/IBM/ibm-common-service-operator/pull/2467/files#diff-9ca5168f345ea4b858cc47f030a07f00e7aad3308328cab8a1bd59128a005924L472) given [this resource does not have `.metadata.annotations.version` defined](https://github.com/IBM/ibm-common-service-operator/pull/2467/files#diff-c7e663f6446b3e86fb9afec8f74ab25b8afe1c145e99baa6389f7a0bf508c38bR75).

Can you have a test on CS operator without above particular Certificate Kind handling logic, and see if CS operator will update the certificate? Thank you!